.github/workflows/publish.yml

```diff
@@ -60,3 +63,20 @@ jobs:
           config: "./.github/configs/cr.yaml"
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Login to GHCR
+        uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc # v2.2.0
+        with:
+          registry: ghcr.io
+          username: ${ GITHUB_REPOSITORY_OWNER }
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push chart to GHCR
```

Comment:
As we are looking to publish the charts as OCI artifacts, I am wondering is there is also an interest in signing the charts using Sigstore and `cosign`. Other aspects of the Argo ecosystem are signing content and it would be good if we were able to reuse a lot of the same paradigms here.

The following is a snippet that illustrates how to not showcase how to publish the charts as OCI artifacts, but also how to sign them using Sigstore

https://github.com/sigstore/helm-charts/blob/main/.github/workflows/release.yml#L73-L83

Reply:
I thought about the same but wanted to create a separate PR for  it! For now this PR is to get OCI charts in place

Reply:
Would we want to start with unsigned charts and then move to signed charts or tackle both at the same time. I defer to the project maintainers and community on the thoughts of the collective group

Reply:
I am fine with booth! If the maintainers want it added into this PR happy to do so.

Reply:
I think we can skip cosign for now. We should clarify with  argoproj maintaines what key to use for this.

But can we use the codeblock from cosign mentioned above? Your PR have two issues
- `find` needs to filter for tgz only as we also generate the provenance file for the old non-OCI index
- `/charts` doesn't exists as a repo in github.com/argoproj

Reply:
Hey @mkilchhofer,

- Done for filtering only the *.tgz! Thanks for the hint!

- /charts will be created and is part of the name. The finale OCI artefact name would look like this e.g: `oci://ghcr.io/argo-helm/charts/argo-cd`

Reply:
I think the first element after
ghci.io would be `argoproj` then.

I prefer having `ghcr.io/argoproj/argo-helm` for consitency:

```
helm push "${pkg}" oci://ghcr.io/${GITHUB_REPOSITORY}
```

Reply:
Sounds good! Changed it.